entries/v/vanguard.com.json

```
@@ -13,7 +13,7 @@
       "Voice Verification"
     ],
     "documentation": "https://investor.vanguard.com/security/security-keys",
-    "notes": "Hardware based 2FA requires SMS / Phone call 2FA as a backup.",
+    "notes": "Hardware based 2FA requires two separate security keys or SMS / Phone call 2FA as a backup. Security keys can not be used to log in to mobile app or mobile website unless \"Preview as Desktop\" is enabled.",
```

As the requirement of 2 keys is a common standard, it doesn't warrant a note.
The switching of user-agent by using "Preview as Desktop" is a bit of a hack and would be better left out as it's not an officially supported mode.
```suggestion
    "notes": "Security keys can not be used to log in to mobile app or mobile website.",
```